mopidy/config/types.py

```diff
@@ -283,12 +411,9 @@ def __init__(self, choices=None, optional=False):
         )
 
 
-class _ExpandedPath(str):
-    def __new__(cls, original, expanded):
-        return super().__new__(cls, expanded)
-
-    def __init__(self, original, expanded):
-        self.original = original
+# Keep this for backwards compatibility
+class _ExpandedPath(_TransformedValue):
+    pass
```

Comment:
Do you know of any extensions using it? Given that is prefixed with an underscore, I'd feel quite okay with removing it.

Reply:
Only the built-in file extension. It was easier to leave it in place rather than delete it, because it gets revamped in this commit I'll be submitting as a follow-up PR:

https://github.com/mopidy/mopidy/commit/07081a7170c4e94c0268972725ffb279b9f2c438